app/views/admin/invoices/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><center>Admin Invoices Index </center></h1>
+<hr>
+<br>
+<% @invoices.each do |invoice|%>
+  <p> Invoice ID: <%= link_to "#{invoice.id}", admin_invoice_path(invoice) %> </p>
```

Comment:
nice use of `path`! this should be something we do across the entire project when refactoring!